src/guiguts/highlight.py

```diff
@@ -433,3 +441,36 @@ def remove_highlights_quotbrac() -> None:
         HighlightTag.CURLY_SINGLE_QUOTE,
     ):
         maintext().tag_delete(tag)
+
+
+def highlight_aligncol_callback(value: bool) -> None:
+    """Callback when highlight_aligncol active state is changed."""
+    if value:
+        root().aligncol = maintext().get_insert_index().col
+        highlight_aligncol()
+    else:
+        remove_highlights_aligncol()
+
+
+def highlight_aligncol() -> None:
+    """Add a highlight to all characters in the alignment column."""
+    if root().highlight_aligncol.get():
+        (top_index, bot_index) = get_screen_window_coordinates()
+
+        col = root().aligncol
+        row = int(top_index.split(".")[0])
+        end_row = int(bot_index.split(".")[0])
+
+        maintext().tag_delete(HighlightTag.ALIGNCOL)
+        _highlight_configure_tag(HighlightTag.ALIGNCOL, HighlightColors.ALIGNCOL)
+        while row <= end_row:
+            # find length of row; don't highlight if row is too short to contain col
```

Comment:
Just curious -- would a space fill be necessary to highlight the column if the row is shorter than the position where the cursor is?

Reply:
I couldn't find a way to highlight past the end of the line (without highlighting the entire rest of the line up until the beginning of the next line). So yes, I believe the only way to do it would be to add whitespace.

Reply:
I found the same with GG1 - it's just not possible with Tk. I don't think we should add whitespace for that purpose - things like search/replace would find that whitespace, and goodness knows what other complications it would introduce.